src/pkg/build/builder_remote.go

```diff
@@ -99,6 +99,11 @@ func (rb *RemoteBuilder) Build(ctx context.Context) (err error) {
 		return err
 	}
 
+	fmt.Printf("\t+-> Build job submited with ID:\t%v\n", rd.ID.Hex())
+
+	fmt.Printf("Once your build has completed, your image will be available at %v\nOr you can pull it with the CLI by running\nsingularity pull --library-uri %v library://%v/build/image",
+		rd.LibraryURL, rd.LibraryURL, rd.ID.Hex())
```

Comment:
This `fmt.Printf` doesn't have a trailing newline, so the command prompt is concatenated at the end of the line:

```sh
adam@ubuntu18:~/go/src/github.com/singularityware/singularity$ ./builddir/singularity build --remote -d /tmp/image docker://busybox
	+-> Build job submited with ID:	5b21d1b2b56ece00012271ec
Once your build has completed, your image will be available at https://library-test.sylabs.io
Or you can pull it with the CLI by running
singularity pull --library-uri https://library-test.sylabs.io library://5b21d1b2b56ece00012271ec/build/imageadam@ubuntu18:~/go/src/github.com/singularityware/singularity$ 
```

I think the HTTP link should be to the image itself (https://library-test.sylabs.io/library/rb-5b21d1b2b56ece00012271ec/build/image) instead of the base of the Library (https://library-test.sylabs.io.)

The `library://` link is not correct. Rather than attempting to build it up yourself using `rd.ID.Hex()`, you should use `rd.LibraryRef`.

On the formatting front, how about:

```
adam@ubuntu18:~/go/src/github.com/singularityware/singularity$ ./builddir/singularity build --remote -d /tmp/image docker://busybox
Build submitted! Once it is complete, the image can be retrieved by running:

    singularity pull --library-uri https://library-test.sylabs.io library://rb-5b21d1b2b56ece00012271ec/build/image

Alternatively, you can access it with a browser at https://library-test.sylabs.io/library/rb-5b21d1b2b56ece00012271ec/build/image
adam@ubuntu18:~/go/src/github.com/singularityware/singularity$ 
```

And finally, I think this should be printed if the build is detached. An attached build should function just like a local build... and it's confusing to then have the output talking about `pull` when this is going to happen automatically.

Reply:
Changes made @tri-adam 
```
#Without LibRef on the args
$ ./builddir/singularity --tokenfile /tmp/sylabs-token build --remote -d --remote-url server:5050 /tmp/image docker://alpine
Build submitted! Once it is complete, the image can be retrieved by running:

        singularity pull --library-uri https://library-test.sylabs.io library://rb-5b228dd73b3b3600018ce527/build/image

Alternatively, you can access it from a browser at https://library-test.sylabs.io/library/rb-5b228dd73b3b3600018ce527/build/image
```
```
#With LibRef on the args
$ ./builddir/singularity --tokenfile /tmp/sylabs-token build --remote -d --remote-url server:5050 library://dummy/builds/detached:test5 docker://alpine
Build submitted! Once it is complete, the image can be retrieved by running:

        singularity pull --library-uri https://library-test.sylabs.io library://dummy/builds/detached:test5

Alternatively, you can access it from a browser at https://library-test.sylabs.io/library/dummy/builds/detached:test5
```